File: src/test/samples/validSamples/validSample.dart
//#region FirstRegion
int x = 42;
// #endregion

// #region Second Region
class MyClass {
  // #region InnerRegion
  void myMethod() {}
  //#endregion

//   #region
    void myMethod2() {}
  //      #endregion ends unnamed region
}
// #endregion

File: src/test/samples/validSamples/validSample.kt
fun main() {
    // #region FirstRegion
    val x = 42
    //#endregion

    //#region Second Region  
    class MyClass {
        // #region   InnerRegion   
        fun myMethod() {}
        //  #endregion   ends InnerRegion 

        // #region
        fun myMethod2() {}
               //#endregion 
    }
    // #endregion
}
